frontend/cli/cmd_interactive.go

```diff
@@ -9,19 +9,28 @@ import (
 
 	"github.com/alecthomas/kong"
 	"github.com/chzyer/readline"
+	kongcompletion "github.com/jotaen/kong-completion"
 	"github.com/kballard/go-shellquote"
+	"github.com/posener/complete"
 
 	"github.com/TBD54566975/ftl/internal/projectconfig"
 )
 
+var _ readline.AutoCompleter = &FTLCompletion{}
+
 type interactiveCmd struct {
 }
 
-func (i *interactiveCmd) Run(ctx context.Context, k *kong.Kong, projectConfig projectconfig.Config) error {
+type FTLCompletion struct {
```

Comment:
I know I said to put all the types at the top, but I'm going to slightly contradict myself slightly 😂

It's preferable to have type followed immediately by methods. So in this case I'd put FTLCompletion + methods at the top, then interactiveCmd + methods below that, so it still works to read down the file and have seen everything.

Not a big deal though, but just for your reference.

Reply:
That is where I was going to put it, but I remembered your comment from yesterday :-) I will move it.